icons/prefect.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Prefect</title><path d="M159.608 765.452c.5-.296 219.453-106.435 219.638-106.545 2.996-1.849 4.697-2.293 4.789-7.564l-.092-413.179c-92.767 46.384-223.336 111.52-234.618 117.512-3.587 1.996-4.088 3.698-4.088 6.288v392.818c-.092 8.878.5 17.643 14.37 10.672zm463.151-641.695v234.008S177.047 134.817 161.624 126.847c-10.283-5.381-10.283-10.486 0-16.256 9.47-5.289 190.324-95.467 218.048-109.338 3.587-1.591 4.697-1.591 7.879-.296l230.734 115.608c3.292 1.608 4.291 3.199 4.476 7.195zm-30.626 218.639L383.851 238.163v26.54l208.282 77.694zm30.626 15.461L383.85 238.163v243.291zm-208.486 107.84l-30.422-180.855v196.613z"/></svg>
```

Comment:
When I [view the SVG on GitHub](https://user-images.githubusercontent.com/3742559/127745685-f945ae9e-845a-4469-bcc5-7b0e839b139c.png) I'm seeing nothing. If I open the SVG in Inkscape it [reveals that the path is entirely outside the viewbox](https://user-images.githubusercontent.com/3742559/127745701-850d5667-a2eb-4084-922c-46a740ad0345.png). Before we can review the SVG you have to make sure it fits precisely into the viewbox - in this case touching the top and bottom edges and centered horizontally.